app/models/profile.rb

```diff
@@ -84,6 +84,13 @@ def supported_os_versions
   private
 
   def bm_versions
+    # rubocop:disable Rails/Output
+    if attributes['bm_versions']
+      puts '---Reached cached attributes in bm_versions---'
+    else
+      puts '---No cached attributes in bm_versions---'
```

Comment:
```suggestion
      `Rails.logger.info` '---No cached attributes in bm_versions---'
```

Reply:
Should this be like Rails.logger.info(...) or did you mean to add the `` around Rails.logger.info?

Reply:
no `` :see_no_evil: typo

Reply:
Ok I figured 😄